packages/create-expo/src/Template.ts

```diff
@@ -201,6 +201,13 @@ export const defaultRenameConfig = [
   'ios/Podfile',
   'ios/**/*.xcodeproj/project.pbxproj',
   'ios/**/*.xcodeproj/xcshareddata/xcschemes/*.xcscheme',
+  'ios/**/*.xcworkspace/contents.xcworkspacedata',
+
+  // macOS
+  'macos/Podfile',
```

Comment:
These changes are probably fine to land without tests.

Reply:
Got it, I've pushed that in the commit `chore(cli,create-expo): revert changes to fixtures and tests` now 👌